R/module_tabs_with_filters.R

```diff
@@ -14,44 +14,76 @@
 #'
 #' @inheritParams module_teal
 #'
-#' @param datasets (`named list` of `FilteredData`)\cr
+#' @param datasets (named `list` of `FilteredData`)
 #'   object to store filter state and filtered datasets, shared across modules. For more
 #'   details see [`teal.slice::FilteredData`]. Structure of the list must be the same as structure
 #'   of the `modules` argument and list names must correspond to the labels in `modules`.
 #'   When filter is not module-specific then list contains the same object in all elements.
 #' @param reporter (`Reporter`) object from `teal.reporter`
 #'
 #' @return A `tagList` of The main menu, place holders for filters and
-#'   place holders for the teal modules
+#'   place holders for the `teal` modules.
```

Comment:
```suggestion
#' @return A `tagList` containing the main menu, placeholders for filters and placeholders for the `teal` modules.
```

Reply:
```suggestion
#' @return A `tagList` containing the main menu, placeholders for filters, and placeholders for the `teal` modules.
```

Reply:
Note that we use `shiny.tag.list` above on parameters and elsewhere as a return.

Slightly in favor of `shiny.tag.list`. Only slightly though, as pragmatically, `tagList` is more verbose.

Reply:
`shiny.tag.list`